run/README.physics_files

```diff
@@ -154,6 +154,8 @@ Other Options:
 
 Urban Physics (sf_urban_physics = 1, 2, and 3)
 	URBPARM.TBL
+	(if using WUDAPT LCZ use_wudapt_lcz - 1)
```

Comment:
@kkeene44 
should this be "use_wudapt_lcz = 1" instead of "use_wudapt_lcz - 1"

Reply:
@davegill It absolutely should. Thanks! I've changed it.